DEPENDENCIES/ML-HElib/src/helayers/hebase/HelayersTimer.cpp

```diff
@@ -0,0 +1,325 @@
+/*
+* MIT License
+*
+* Copyright (c) 2020 International Business Machines
+*
+* Permission is hereby granted, free of charge, to any person obtaining a copy
+* of this software and associated documentation files (the "Software"), to deal
+* in the Software without restriction, including without limitation the rights
+* to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+* copies of the Software, and to permit persons to whom the Software is
+* furnished to do so, subject to the following conditions:
+*
+* The above copyright notice and this permission notice shall be included in all
+* copies or substantial portions of the Software.
+*
+* THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+* IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+* FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+* AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+* LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+* OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+* SOFTWARE.
+*/
+
+#include "HelayersTimer.h"
+#include "AlwaysAssert.h"
+#include <iostream>
+#include <iomanip>
+#include <mutex>
+#include <math.h>
+#include <time.h>
+#include <omp.h>
+
+#ifdef _WIN32
+#include <windows.h>
+#endif
+
+using namespace std::chrono;
+using namespace std;
```

Comment:
For code clarity and prevent ambiguity, I would recommend I would avoid `using namespace xxx` and explicitly call the functions using its fully qualified name, e.g `std::cout`.

Reply:
github issue will be created to revisit Namespace directive style choices.